hw23/js/script.js

```diff
@@ -0,0 +1,76 @@
+import {
+  Validator,
+  isNotEmpty,
+  minMaxLength,
+  isEmail,
+  isPassword,
+  confirmPassword,
+} from "./validator.js";
+
+const formConfigs = {
+  "enter-username": [isNotEmpty, minMaxLength],
+  "enter-email": [isNotEmpty, isEmail],
+  "enter-password": [isNotEmpty, isPassword],
+  "reenter-password": [isNotEmpty, confirmPassword],
+};
+
```

Comment:
Please refactor follow code. It's my quick-hand code and it should be refactored to get rid of duplicates